### PR TITLE
Create Windows.Forensics.Jumplists_JLECmd

### PR DESCRIPTION
**Pull request by orhan-emre**
- This artifact executes Eric Zimmerman's JLECmd to parse AUTOMATICDESTINATIONS-MS and CUSTOMDESTINATIONS-MS files in C:\ drive recursively and return output for analysis. (jlecmd.exe -d C:/ --csvf -csv tmpdir results.csv).  
- JLECmd.zip is downloaded from the URL to 'C:\Program Files\Velociraptor\Tools' folder.    
- JLECmd.zip can be uploaded to Velociraptor Server in order to copy it to the clients in case there is no internet connection.
- This artifact is created using @carlos_cajigas LECmd VQL as a quide.  
- JLECmd is a CLI tool for analyzing Custom Destinations jump list data. Learn more  https://github.com/EricZimmerman/JLECmd

### CODE REVIEW

File: content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    * JLECmd.zip is downloaded from the URL to 'C:\Program Files\Velociraptor\Tools' folder.   
+    * JLECmd.zip can be uploaded to Velociraptor Server in order to copy it to the clients in case there is no internet connection.
+    * Created using @carlos_cajigas LECmd VQL as a quide.  
+    * JLECmd is a CLI tool for analyzing Custom Destinations jump list data. Learn more - https://github.com/EricZimmerman/JLECmd
+
+author: Orhan Emre @orhan_emre
+
+type: CLIENT
+
+tools:
```
Comment (mgreen27) [inline content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd]: Can you add an expected hash and version to the tool definition for this artifact ?

Here is an example of another tool: https://github.com/Velocidex/velociraptor-docs/pull/789/files#diff-c34530120068a449fa566d79550de2caf5a5feb479b8392473a0b18ac20ea8a6R17

Comment (orhan-emre) [reply content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd:13]: Thank you for the feedback. I have added the SHA256 hash value and the version of the tool.

**Review by orhan-emre — COMMENTED** [review]
Thank you for the feedback. I have added the SHA256 hash value and the version of the tool

File: content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  - name: JLECmd
+    url: https://f001.backblazeb2.com/file/EricZimmermanTools/JLECmd.zip
+    expected_hash: 10bc3698867be7c707500afc01c45c44b6a0439d14e04477e08b76d95c87255c
+    version: 1.5.0
```
Comment (orhan-emre) [inline content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd]: I have added SHA256 hash and version to the tool definition.

File: content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd (hunk 1 above)
Comment (orhan-emre) [reply content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd]: Thank you for the feedback. I have added the SHA256 hash value and the version of the tool

File: content/exchange/artifacts/Windows.Forensics.Jumplists_JLECmd.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        (if(condition=dateAfter, then=SourceCreated > dateAfter,
+            else=TRUE) AND 
+        if(condition=dateBefore, then=SourceCreated < dateBefore, 
+            else=TRUE))
+      AND SourceFile =~ sourceFile
+      AND LocalPath =~ localPath
+      AND Arguments =~ arguments
+
+
+  - query: |
```
Comment (orhan-emre) [inline content/exchange/artifacts/Windows.Forensics.Jumplists_JLECmd.yaml:80]: this query has been added in order to merge with 793.

File: content/exchange/artifacts/Windows.Forensics.Jumplists_JLECmd.yaml [hunk 4]
```diff
@@ -0,0 +1,95 @@
+name: Windows.Forensics.Jumplists_JLECmd
+description: |
```
Comment (orhan-emre) [inline content/exchange/artifacts/Windows.Forensics.Jumplists_JLECmd.yaml:2]: The description and the name have been updated.

**Comment by orhan-emre** [comment]
793 is merged with 792 by adding additional query.

**Pull request merged**